blog/2021-03-16-BatchSolvingWithAmq.adoc

```diff
@@ -0,0 +1,182 @@
+= Batch solving in a stateless manner with OptaPlanner and AMQ
+:page-interpolate: true
+:imagesdir: 2021/03/16
+:awestruct-author: rsynek
+:awestruct-layout: blogPostBase
+:awestruct-tags: [amq, quarkus, messaging, integration]
+:awestruct-share_image_filename: architecture.png
+
+Let's take a look at a new https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/amq-quarkus-school-timetabling[quickstart], which shows how to build a scalable, stateless service capable of solving numerous instances of the same planning problem.
```

Comment:
The first paragraph is very important, almost as important as the title. It should tell poeple what it's solving - why it's worth their time.

Maybe I'd reverse the sentence: something like: 

```suggestion
If I want to solve many different datasets of a planning problem every night, what architecture can easily scale out horizontally without loss of data? In this article, we'll take a look at how to put a transactional AMQ queue in front of a set of stateless OptaPlanner pods, so other applications can submit datasets to solve and listen to the resulting solutions without having to worry about which OptaPlanner pod does the actual solving.
```

Reply:
I slightly reworded the suggestion but used most of it. Thanks!